package.json

```diff
@@ -13,7 +13,7 @@
     "lerna:bootstrap": "lerna bootstrap",
     "lerna:publish": "yarn test && lerna publish",
     "lint": "eslint --ext .js --ignore-path .gitignore .",
-    "test": "jest"
+    "test": "env TZ='Etc/UTC' jest"
```

Comment:
I don't think this library should depend on the system timezone. If we cannot avoid this ENV dependency, can we proactively set it when initializing the lib?

Reply:
When Chrono builds the JavaScript Date object from its parsed component, the Date object will have the system timezone.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/Date

So they try to compensate that by getting the current system timezone
https://github.com/wanasit/chrono/blob/0f1620418a4becba6a4e6ff1c743da8bb411aea7/src/results.ts#L143

This create some funky situation like this, the result should be 08:00+08:00 not 07:00+08:00
![image](https://user-images.githubusercontent.com/11194590/129345351-1f0b56db-3655-4651-ac77-02a682e18b9d.png)

Didn't have time to report to them yet

Reply:
If they fix this bug, I think we then can remove this env

Reply:
Will the library users need to do anything about this? Do they need to config the system timezone to be able to use this library properly?

Reply:
Yes, the users of this library should always set their NodeJS system timezone as "Etc/UTC" to avoid potential bugs.
I'll add this to our README

Also reported this bug to Chrono.
https://github.com/wanasit/chrono/issues/403

Reply:
hmm that is not great because this library is supposed to be used on browsers as well. Holistics is also using this on browsers